src/cascadia/WindowsTerminal/WindowThread.cpp

```diff
@@ -191,6 +193,18 @@ int WindowThread::_messagePump()
             break;
         }
 
+        if (!_loggedInteraction && (message.message == WM_KEYDOWN || message.message == WM_SYSKEYDOWN))
+        {
+            TraceLoggingWrite(
+                g_hWindowsTerminalProvider,
+                "InteractiveSession",
+                TraceLoggingDescription("Emmitted the first time a key is pressed"),
+                TraceLoggingLevel(WINEVENT_LEVEL_VERBOSE),
+                TraceLoggingKeyword(TIL_KEYWORD_TRACE),

```

Comment:
Drop this - it may cause the data pipeline to ignore it. TIL_KEYWORD_TRACE is only for high-volume _non-ProductAndServiceUsage_ events.

Reply:
TIL! 😉